src/App.js

```diff
@@ -4,15 +4,67 @@ import SignUpPage from "./Components/Pages/SignUpPage";
 import LoginPage from "./Components/Pages/LoginPage";
 import ItemDetailsPage from "./Components/Pages/ItemDetailsPage";
 
+const dummyData = [
+  {
+    id: "1",
+    title: "Isaac Asimov: The Complete Stories, Vol. 1",
+    description:
+      "Isaac Asimov best work In a world of robots how can we keep on keeping on",
+    price: 68.9,
+    img: "isaac_asimov.jpeg",
+  },
+  {
+    id: "2",
+    title: "Isaac Asimov: The Complete Stories, Vol. 1",
+    description:
+      "Isaac Asimov best work In a world of robots how can we keep on keeping on",
+    price: 68.9,
+    img: "isaac_asimov.jpeg",
+  },
+  {
+    id: "3",
+    title: "Isaac Asimov: The Complete Stories, Vol. 1",
+    description:
+      "Isaac Asimov best work In a world of robots how can we keep on keeping on",
+    price: 68.9,
+    img: "isaac_asimov.jpeg",
+  },
+  {
+    id: "4",
+    title: "Isaac Asimov: The Complete Stories, Vol. 1",
+    description:
+      "Isaac Asimov best work In a world of robots how can we keep on keeping on",
+    price: 68.9,
+    img: "isaac_asimov.jpeg",
+  },
+  {
+    id: "5",
+    title: "Isaac Asimov: The Complete Stories, Vol. 1",
+    description:
+      "Isaac Asimov best work In a world of robots how can we keep on keeping on",
+    price: 68.9,
+    img: "isaac_asimov.jpeg",
+  },
```

Comment:
Duplicate data. There should be more than 2 options... 😦

Reply:
Yes, this is only to showcase what the front-end looks like with data.